internal/controller/supervisorconfig/githubupstreamwatcher/github_upstream_watcher_test.go

```diff
@@ -22,7 +22,7 @@ import (
 	"go.pinniped.dev/internal/federationdomain/dynamicupstreamprovider"
 	"go.pinniped.dev/internal/federationdomain/upstreamprovider"
 	"go.pinniped.dev/internal/plog"
-	"go.pinniped.dev/internal/testutil/githubtestutil"
+	"go.pinniped.dev/internal/testutil/oidctestutil"
```

Comment:
this seems odd, why are `GitHub` items imported through `oidc`?

Reply:
Yeah, because all kinds of "not OIDC" utils are here.
It is the right move to keep them together, but we should probably break these apart in a future cleanup PR.